controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```
@@ -204,67 +202,20 @@ func (r *AKODeploymentConfigReconciler) reconcileCloudUsableNetwork(
 	log logr.Logger,
 	obj *akoov1alpha1.AKODeploymentConfig,
 ) (ctrl.Result, error) {
-	res := ctrl.Result{}
-
 	log = log.WithValues("cloud", obj.Spec.CloudName)
-	log.Info("Start reconciling AVI cloud usable networks")
-
-	requeueAfter := ctrl.Result{
-		Requeue:      true,
-		RequeueAfter: time.Second * 60,
-	}
-
-	network, err := r.aviClient.NetworkGetByName(obj.Spec.DataNetwork.Name)
-	if err != nil {
-		log.Error(errors.Errorf("[WARN]Failed to get the Data Network %s from AVI Controller", obj.Spec.DataNetwork.Name), "")
-		return requeueAfter, nil
-	}
+	log.Info("Start reconciling AVI cloud usable network")
 
-	cloud, err := r.aviClient.CloudGetByName(obj.Spec.CloudName)
+	added, err := r.AddUsableNetwork(r.aviClient, obj.Spec.CloudName, obj.Spec.DataNetwork.Name)
 	if err != nil {
-		log.Error(err, "Faild to find cloud, requeue the request")
-		// Cannot find the configured cloud, requeue the request but
-		// leave enough time for operators to resolve this issue
-		return requeueAfter, nil
+		log.Error(err, "Failed to add usable network", obj.Spec.DataNetwork.Name)
+		return ctrl.Result{Requeue: true, RequeueAfter: time.Minute}, nil
```

You can just return the err